client/blocks/reader-full-post/style.scss

```diff
@@ -412,6 +412,24 @@
 	margin: 28px 0 26px;
 }
 
+.reader-full-post .embed-youtube,
+.reader-full-post .embed-vimeo {
+	display: block;
+	position: relative;
+	padding: 25px 0 56.25%;
+
+	iframe {
+		height: 100%;
```

Comment:
kk. we could probably land this to fix up youtube and vimeo for now, while we work out a js solution?